.github/workflows/ci.yml

```diff
@@ -33,7 +31,29 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  changed:
+    name: Get changed files
+    runs-on: macos-14
```

Comment:
Should we use `ubuntu-latest` as it's usually faster? (and cheaper cost-wise even though we don't pay for it)

Reply:
In my experience `macos-14` is faster because it uses m1. I am fine using ubuntu if the team has problems with `macos-14`

Reply:
I think it's more or less on par and not always fastest. Our CI runs have ubuntu sometimes running faster too 🤔